autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -626,6 +822,22 @@ def send_warnings_to_log(message, category, filename, lineno,
 
     def _predict_regression(self, X: np.ndarray, pipeline: BaseEstimator,
                             Y_train: Optional[np.ndarray] = None) -> np.ndarray:
+        """
+        A wrapper function to handle the prediction of regression tasks.
+        It is a wrapper to provide the same interface to _predict_proba
+
+        Regression predictions expects an unraveled dimensionality.
```

Comment:
Could you give an example?
For example, when you have the predictions whose shape is `(N,)`, should it be either `(N, 1)` or `(N, )`

Reply:
Thanks, I have improved the docstring with this!